strawberry_django/ordering.py

```diff
@@ -51,17 +64,42 @@ def generate_order_args(order: WithStrawberryObjectDefinition, prefix: str = "")
         elif ordering == Ordering.DESC:
             args.append(f"-{prefix}{f.name}")
         else:
-            subargs = generate_order_args(ordering, prefix=f"{prefix}{f.name}__")
+            subargs = generate_order_args(
+                ordering,
+                prefix=f"{prefix}{f.name}__",
+                sequence=(_seq := sequence.get(f.name)) and _seq[1],
+            )
             args.extend(subargs)
 
     return args
 
 
-def apply(order: Optional[WithStrawberryObjectDefinition], queryset: _QS) -> _QS:
+def apply(
+    order: Optional[WithStrawberryObjectDefinition],
+    queryset: _QS,
+    info: Optional[Info] = None,
+) -> _QS:
     if order in (None, strawberry.UNSET):  # noqa: PLR6201
         return queryset
 
-    args = generate_order_args(order)
+    sequence: _OrderSequence = {}
+    if info is not None and info._raw_info.field_nodes:
+        field_node = info._raw_info.field_nodes[0]
```

Comment:
we should abstract this in strawberry, let's remember to chat about this in future 😊